test/test_connection.py

```diff
@@ -153,6 +150,14 @@ def test_ping_pong(self, as_client):
             data = wire_data[2:]
         assert data == payload
 
+        # Verify that connection vends the PongReceive event with the correct payload
```

Comment:
I'm not trying to be argumentative or anything, I just (still) have no idea what the word is supposed to mean in this context. Did I somehow miss something? It doesn't seem to show up anywhere in the h11, h2, or wsproto source trees...